docs/source/gateway/index.rst

```diff
@@ -623,18 +630,91 @@ Once the MLflow AI Gateway server has been configured and started, it is ready t
 Standard Query Parameters
 -------------------------
 
-The query parameters that are supported by various providers for different route types are also available to be used with the MLflow AI Gateway.
-Each of these query parameters are optional elements that can be included along with using the ``query`` APIs as key value pairs within the ``data`` argument.
-The AI Gateway will perform validation of these commonly modified parameters to ensure that provider-specific restrictions and scaling factors are unified with a consistent range of allowable values.
-If a given provider does not provide support for a parameter, a clear message will be returned when queried that explains the restrictions for the given provider and route type.
-
-- ``temperature``: This parameter controls the randomness of predictions by scaling the logits before applying softmax. A value closer to 0.0 makes the output more deterministic, while a value closer to 1.0 makes it more diverse. Default is 0.0.
-
-- ``max_tokens``: This parameter limits the length of the generated output by specifying a maximum token count. The range is from 1 to infinity, and by default, there is no limit (infinity). Some providers have a maximum value associated with this parameter that the AI Gateway will enforce to prevent a provider-generated exception.
+The MLflow AI Gateway defines standard parameters for chat, completions, and embeddings that can be
+used when querying any route regardless of its provider. Each parameter has a standard range and
+default value. When querying a route with a particular provider, the MLflow AI Gateway automatically
+scales parameter values according to the provider's value ranges for that parameter.
```

Comment:
An example might clarify what this means. For example, we could say "For example, the temperature will be multiplied by 2 for OpenAI".